Sources/Orders/Orders.docc/GettingStarted.md

```diff
@@ -9,21 +9,23 @@ For all the other custom data needed to generate the order, such as the barcodes
 The order data model will be used to generate the `order.json` file contents.
 
 The order you distribute to a user is a signed bundle that contains the `order.json` file, images, and optional localizations.
-The Orders framework provides the ``OrdersService`` class that handles the creation of the order JSON file and the signing of the order bundle, using an ``OrdersDelegate`` that you must implement.
+The Orders framework provides the ``OrdersService`` class that handles the creation of the order JSON file and the signing of the order bundle.
 The ``OrdersService`` class also provides methods to send push notifications to all devices registered when you update an order, and all the routes that Apple Wallet uses to retrieve orders.
 
 ### Implement the Order Data Model
 
-Your data model should contain all the fields that you store for your order, as well as a foreign key to ``Order``, the order model offered by the Orders framework.
+Your data model should contain all the fields that you store for your order, as well as a foreign key to ``Order``, the order model offered by the Orders framework, and a order type identifier that's registered with Apple.
 
 ```swift
 import Fluent
-import struct Foundation.UUID
+import Foundation
 import Orders
 
 final class OrderData: OrderDataModel, @unchecked Sendable {
     static let schema = "order_data"
 
+    static let typeIdentifier = Environment.get("ORDER_TYPE_IDENTIFIER")!
```

Comment:
Is this best practice?

Reply:
Personally I prefer not to hard code it so as not to leak it into version control, just to be safe. It's not actually strictly secret, so you might as well hard code it.